### PR TITLE
Fix labels, modal refresh, policy editor default tags

### PR DESCRIPTION
**Pull request by saminzadeh**

### CODE REVIEW

File: ui/src/components/RollupRuleEditor.js [hunk 1]
```diff
@@ -123,21 +134,27 @@ const TargetsEditorBase = props => {
                 <Icon type="delete" />
               </a>
             }>
-            <label>Metric Name</label>
+            <label>New Metric Name</label>
```
Comment (xichen2020) [inline ui/src/components/RollupRuleEditor.js]: Do you want to call this `New Rollup Metric Name`?

File: ui/src/components/RollupRulesTable.js [hunk 2]
```diff
@@ -38,7 +38,7 @@ function TargetPreview({target}) {
         <div>{target.policies.join(' ')}</div>
       </div>
       <div className="pb1">
-        <label className="bold pb1">Tags</label>
+        <label className="bold pb1">GroupBy Tags</label>
```
Comment (xichen2020) [inline ui/src/components/RollupRulesTable.js:41]: Let's still call this `Rollup Tags` for now and see whether the user finds it confusing.

Comment (saminzadeh) [reply ui/src/components/RollupRulesTable.js:41]: sg

File: ui/src/components/RollupRuleEditor.js (hunk 1 above)
Comment (saminzadeh) [reply ui/src/components/RollupRuleEditor.js]: Sure, that works. Might be worth to user test these types of things. It's hard to say with our tunnel vision now lol

Comment (xichen2020) [reply ui/src/components/RollupRuleEditor.js]: Agreed =p

**Review by xichen2020 — APPROVED** [review]
LGTM

**Pull request merged**